iamproperty/iamproperty.github.io
# Ds 119/snapshot component

## Summary of Changes:
1. Snapshot component

## Rationale
Create a component to use for dashboards

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/snapshot

## Ticket(s)
DS-119
BAU-2220

## Checklist

## Notes
...
